Dockerfile

```diff
@@ -1,14 +1,23 @@
-FROM nodesource/node:trusty
+FROM nodesource/trusty:4.4.7
```

Comment:
Do we want to pin to a particular nodesource version? Would 'latest' work here, or 5?

https://hub.docker.com/r/nodesource/trusty/